step.yml

```
@@ -44,3 +44,9 @@ inputs:
     value_options:
     - "true"
     - "false"
+- derived_data_path: "~/Library/Developer/Xcode/DerivedData/**"
```

Is there a reason to cache more than the SourcePackages subdirectory by default since that is what the step was doing originally?

```suggestion
- derived_data_path: "~/Library/Developer/Xcode/DerivedData/**/SourcePackages"
```

---

Hi @liamnichols the new step input (`derived_data_path `) control the file path of the DerivedData directory and later on the step construct the SourcePackages path by joining `SourcePackages` to the given DerivedData path:  https://github.com/bitrise-steplib/bitrise-step-save-spm-cache/pull/6/files#diff-f9b80317077cdd80a92e3996a30b0f49975f2bdb00b194cf4bab3538a0850654R63